File: ElectronicsShop.java
// https://www.hackerrank.com/challenges/electronics-shop/problem?isFullScreen=true

class Electronic{
     static int getMoneySpent(int[] keyboards, int[] drives, int b) {
        int current = 0;
        int max = -1;
        for(int i = 0; i < keyboards.length; i++){
            for(int j = 0; j < drives.length; j++){
                if(keyboards[i] + drives[j] <= b){
                    current = keyboards[i] + drives[j];
                    max = Math.max(max, current);
                }
            }
        }
        return max;
    }

    public static void main(String[] args) {
        int[] keyboards = {40, 50, 60};
        int[] drives = {5, 8, 12};
        int b = 60;
        int moneySpent = getMoneySpent(keyboards, drives, b);
        System.out.println("Maximum money spent: " + moneySpent);
    }
}

File: Python/ElectronicsShop.py
# https://www.hackerrank.com/challenges/electronics-shop/problem?isFullScreen=true

class ElectronicsShop:
    def getMoneySpent(self, keyboards, drives, b):
        max_spent = -1
        for keyboard_price in keyboards:
            for drive_price in drives:
                total_price = keyboard_price + drive_price
                if total_price <= b and total_price > max_spent:
                    max_spent = total_price
        return max_spent

if __name__ == "__main__":
    shop = ElectronicsShop()
    keyboards = [40, 50, 60]
    drives = [5, 8, 12]
    b = 60
    result = shop.getMoneySpent(keyboards, drives, b)
    print(f"Maximum money spent: {result}")